src/main.cpp

```diff
@@ -92,6 +94,14 @@ static void lPrintVersion() {
 #endif
 }
 
+
+// Command line argument constants
+static constexpr std::string_view ARG_NAME_ONLY_PREPROCESSOR{"--onlycpp"};
+static constexpr std::string_view ARG_DESC_ONLY_PREPROCESSOR{"Run only the preprocessor (if enabled)"};
+static constexpr std::string_view ARG_NAME_ONLY_PREPROCESSOR_SHORT{"-E"};
+static constexpr std::string_view ARG_DESC_ONLY_PREPROCESSOR_SHORT{"An alias for [--onlycpp]"};
```

Comment:
I think `-E` is sufficient and not need for `--onlycpp`.
In general we prefer to have verbose spelling for the switches, except for very standard ones. And `-E` is this kind of standard switches, that every programmer understands without explanation.

Reply:
I also generally prefer having the verbose spelling. I wasn't familiar with the `-E` switch, so I added the alias, but I can remove it if preferred.

Reply:
yeah, I prefer to remove it. I would put it this way - I would expect everyone who might need it to be familiar with `-E` for C/C++ compilers. And I don't want to have too many switches, just to keep things simple. "Less is more" :)

At the same time, I've talked to engineers who compile a lot of ISPC code through Compiler Explorer and they really like the option to ignore preprocessor failures. I.e. they copy-paste huge file and don't bother if all of the `#include` directives are removed. I suggest adding `--ignore-cpp-errors` or `--ignore-preprocessor-errors`. @JeffRous any opinion on the switch naming?

Reply:
> yeah, I prefer to remove it. I would put it this way - I would expect everyone who might need it to be familiar with `-E` for C/C++ compilers. And I don't want to have too many switches, just to keep things simple. "Less is more" :)
> 
> At the same time, I've talked to engineers who compile a lot of ISPC code through Compiler Explorer and they really like the option to ignore preprocessor failures. I.e. they copy-paste huge file and don't bother if all of the `#include` directives are removed. I suggest adding `--ignore-cpp-errors` or `--ignore-preprocessor-errors`. @JeffRous any opinion on the switch naming?

To clarify, currently `--onlycpp` runs the preprocessor and exits the compiler after it completes. It does not suppress errors (that occur within macro code). Is the behavior you suggest: to add a flag that runs the entire ISPC compile jobs (assuming it can complete) while suppressing errors from `execPreprocessors`?

Reply:
And also remove the alias?

Reply:
I suggest:
- removing the alias, i.e. `--onlycpp`
- add new switch to preserve old behavior of ignoring preprocessor errors. I.e. the compilation should not fail if preprocessor returned an error. Typical use case if not failing if the `#include "my_header.isph"` was not able fide the file to include.

To be clear, these changes are unrelated, I should have mentioned them in two separate comments to avoid confusion.

Reply:
It's resolved, the naming of the "ignore errors" switch is discussed in another thread.